src/test/java/org/example/homework14/DemoTests.java

```diff
@@ -0,0 +1,193 @@
+    private static final By LAST_NAME_FIELD = By.id("lastName");
+    private static final By EMAIL_FIELD = By.id("userEmail");
+    private static final By AGE_FIELD = By.id("age");
+    private static final By SALARY_FIELD = By.id("salary");
+    private static final By DEPARTMENT_FIELD = By.id("department");
+    private static final By SUBMIT_BUTTON = By.id("submit");
+    private static final String FIRST_NAME = "TestName";
+    private static final String LAST_NAME = "Test";
+    private static final String EMAIL = "example@test.com";
+    private static final String AGE = "25";
+    private static final String SALARY = "5000";
+    private static final String DEPARTMENT = "QA";
+    private static final By RAW_LIST = (By.className("rt-tr"));
+
+    @Test
+    @DisplayName("Checks removing of all data in demo table")
+    void checkTableDataRemoving() {
+        WebElement webTablesMenu = driver.findElement(WEB_TABLE_MENU_ITEM);
+        webTablesMenu.click();
+
+        WebElement deleteButton1 = driver.findElement(RAW_DELETE_1_BUTTON);
+        deleteButton1.click();
+
+        WebElement deleteButton2 = driver.findElement(RAW_DELETE_2_BUTTON);
+        deleteButton2.click();
+
+        WebElement deleteButton3 = driver.findElement(RAW_DELETE_3_BUTTON);
+        deleteButton3.click();
+
+        WebElement noTableDataBanner = driver.findElement(NO_TABLE_DATA_ITEM);
+
+        assertTrue(noTableDataBanner.isDisplayed());
+        assertEquals("No rows found", noTableDataBanner.getText());
+    }
+
+    @Test
+    @DisplayName("Checks adding record to demo table")
+    void checkAddTableData() {
+        WebElement webTablesMenu = driver.findElement(WEB_TABLE_MENU_ITEM);
+        webTablesMenu.click();
+
+        WebElement addDataButton = driver.findElement(ADD_DATA_BUTTON);
+        addDataButton.click();
+
+        WebElement firstNameField = driver.findElement(FIRST_NAME_FIELD);
+        firstNameField.sendKeys(FIRST_NAME);
+
+        WebElement lastNameField = driver.findElement(LAST_NAME_FIELD);
+        lastNameField.sendKeys(LAST_NAME);
+
+        WebElement emailField = driver.findElement(EMAIL_FIELD);
+        emailField.sendKeys(EMAIL);
+
+        WebElement ageField = driver.findElement(AGE_FIELD);
+        ageField.sendKeys(AGE);
+
+        WebElement salaryField = driver.findElement(SALARY_FIELD);
+        salaryField.sendKeys(SALARY);
+
+        WebElement departmentField = driver.findElement(DEPARTMENT_FIELD);
+        departmentField.sendKeys(DEPARTMENT);
+
+        WebElement submitButton = driver.findElement(SUBMIT_BUTTON);
+        submitButton.click();
+
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(2));
+        wait.until(webDriver -> true);
+
+        var rawList = driver.findElements(RAW_LIST);
+        List<String> testRaw = Collections.singletonList(rawList.get(4).getText());
+
+        assertEquals(testRaw.get(0), FIRST_NAME);
```

Comment:
Maybe it is not good idea to address by index `get(0)` maybe would be better to check that list contains required text, without pointing to the specific position
Anyway this is disputable, and depends on what exactly do you want to check